packages/glimmer-runtime/tests/attributes-test.ts

```diff
@@ -78,6 +78,39 @@ test("helpers shadow self", () => {
   equalTokens(root, '<div data-test="hello"></div>');
 });
 
+test("disable updates properly", () => {
+  let template = compile('<input disabled={{enabled}} />');
```

Comment:
Can you also add another test just like this one, only with it being quoted?  Basically `<input disabled="{{value}}">` with all the various rerender permutations below.